python/test/gluon/test_lowerings.py

```diff
@@ -119,15 +119,12 @@ def _reduce_layouts():
                                     cta_order=[0, 1], instr_shape=[16, 8]),
         ttgl.NVMMADistributedLayout(version=[3, 0], warps_per_cta=[4, 1], ctas_per_cga=[1, 1], cta_split_num=[1, 1],
                                     cta_order=[1, 0], instr_shape=[16, 16, 16]),
-        ttgl.amd.AMDMFMALayout(version=2, warps_per_cta=[4, 1], tiles_per_warp=[1, 1], instr_shape=[32, 32],
```

Comment:
Why do you only test FMA layouts for versions 3 and 4?

Reply:
Currently we focus on supporting MFMA 3 (for MI300) and MFMA 4 (for MI350).

Reply:
Does that mean you're dropping support for version 2, otherwise it should still be tested right?